### DISCUSSION TITLE
Cannot connect oled display

### DISCUSSION THREAD

**Discussion by rezirezi** [marcel-licence/esp32_basic_synth#105] [Q&A] [answered]
Hi, so i'm trying to connect the screen to see the scope, however when i uncomment #define OLED_OSC_DISP_ENABLED, the synth keeps crashing, I get an error saying "abort() was called at PC 0x400e3109 on core 0" after which it also says "SSD1306 allocation failed
". i tried to find the functions "ScopeOled_Setup()" and ScopeOled_Process() to debug the issue, but I couldn't find them. I could find their definitions in ml_scope.h, but not the functions themselves. is ml_scope.cpp missing perhaps? if anyone could help i'll be very grateful.

**Comment by marcel-licence** [comment, marked as answer]
Hi, which board configuration did you use?
The OLED stuff was precompiled and just linked. I want to change it soon that only the algorithm for the scope is compiled and you can change the code for the output by yourself. It requires some amount of memory.

**Reply by rezirezi** [reply]
Hi, thanks so much for the reply, I have no idea how, but after restarting IDE it magically started working. Should've tried that before posting lol
